assets/sass/_base.sass

```diff
@@ -598,11 +601,11 @@ section
   #vendorStrip
     line-height: 44px
     max-width: 100%
-    overflow-x: auto
     -webkit-overflow-scrolling: touch
 
     ul
       float: none
+      overflow-x: auto
```

Comment:
Is there an advantage to move the `overflow-x: auto`?

Reply:
Yes, because if you leave it as it is, the whole block will move, but we need only the menu

Reply:
👍